doc/setup_instructions.md

```diff
@@ -338,39 +247,79 @@ If it didn't work you can try these:
 to get help. Links to the forums and community discord
 are at the top of this document.
 
-Forking
--------
+Git LFS limits
```

Comment:
`limits` doesn't feel like the right word to use here. `limitations` would be better.